### PR TITLE
[13.0][MIG] partner_employee_quantity

### PR DESCRIPTION
**Pull request by newtratip**
This module allows to set employee quantity for partners, and categorize them by ranges.
Peek 2020-03-18 12-19

### CODE REVIEW

File: partner_employee_quantity/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+# Copyright 2017 David Vidal <david.vidal@tecnativa.com>
+# Copyright 2018 Tecnativa - Pedro M. Baeza
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "Employee quantity in partners",
+    "summary": "Know how many employees a partner has",
+    "version": "13.0.1.0.0",
+    "category": "Customer Relationship Management",
+    "author": "Tecnativa, " "Odoo Community Association (OCA)",
```
Comment (Saran440) [inline partner_employee_quantity/__manifest__.py]:
```suggestion
    "author": "Tecnativa, Odoo Community Association (OCA)",
```

Comment (newtratip) [reply partner_employee_quantity/__manifest__.py]: Fixed, thank for your review :)

**Review by ps-tubtim — APPROVED** [review]
Code LGTM

**Review by lfreeke — APPROVED** [review]
Functional test :+1:

**Review by Saran440 — APPROVED** [review]
Code Review. LGTM :+1:

**Comment by newtratip** [comment]
Hi @NL66278, Could you review/merge this ?

File: partner_employee_quantity/views/res_partner_view.xml [hunk 2]
```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="view_partner_form" model="ir.ui.view">
+        <field name="name">Partner form with employees</field>
+        <field name="model">res.partner</field>
+        <field name="inherit_id" ref="base.view_partner_form" />
+        <field name="arch" type="xml">
+            <page name="internal_notes" position="inside">
+                <group>
```
Comment (NL66278) [inline partner_employee_quantity/views/res_partner_view.xml:9]: Why the two levels of group?

**Review by NL66278 — CHANGES_REQUESTED** [review]
Please make all license lines https:

And a question on the view definition (see other comment).

**Review by HaraldPanten — APPROVED** [review]
Hi @newtratip 

Functional review OK.

Could you attend @NL66278 comments?

File: partner_employee_quantity/views/res_partner_view.xml (hunk 2 above)
Comment (newtratip) [reply partner_employee_quantity/views/res_partner_view.xml:9]: For show these fields to left side of group. You can inherit this module with add some field in right side of group. I think this is beautiful layout.
Selection_543

**Comment by newtratip** [comment]
@NL66278 Edited all license lines to https

**Review by NL66278 — APPROVED** [review]
:+1: LGTM. Might consider enhancement to actually put quantities in the quantity range.

**Pull request merged**